code/src/utils/urls.js

```diff
@@ -0,0 +1,7 @@
+// Api key and urls to fetch the movie list and the details
+
+export const API_KEY = "a42559943824c6a02203973f57136d18";
+export const MOVIES_URL = (API_KEY) =>
+  `https://api.themoviedb.org/3/movie/popular?api_key=${API_KEY}&language=en-US&page=1`;
+export const MOVIE_DETAILS_URL = (API_KEY, movieId) =>
+  `https://api.themoviedb.org/3/movie/${movieId}?api_key=${API_KEY}&language=en-US`;
```

Comment:
A really nice project whit a clean design! everything is working and the code is easy to read! Goood Work!!!!
- [YES ]  Am I able to understand the code easily?
- [NO ]  Does the code have lots of duplicates? Could it be tidied up?
- [ YES it works]  Does it work or does anything look broken?
- [ YES]  Does it follow the general and blue requirements?

Reply:
Tack Jessica for your feedback :)